blocks/library/columns/index.js

```diff
@@ -70,20 +69,11 @@ export const settings = {
 	},
 
 	edit( { attributes, setAttributes, className } ) {
-		const { align, columns } = attributes;
+		const { columns } = attributes;
```

Comment:
I wanted to introduce a similar change in #5099. If you look at https://github.com/WordPress/gutenberg/pull/5099/files#diff-89d3b001ccbb88ada880aa2d5db1c55bR58, `getEditWrapperProps` was also removed because it is handled by hooks.